src/contacts/processors/IEditRecordActionProcessor.java

```diff
@@ -0,0 +1,11 @@
+package contacts.processors;
+
+import contacts.model.Record;
+
+import java.io.IOException;
+
+public interface IEditRecordActionProcessor {
+    void doAction(Record record) throws IOException;
+
+    String getSupportedActionTitle();
```

Comment:
please rename method name to more specific one

Reply:
Done